plugins/test/__snapshots__/main_menu_action.test.jsx.snap

```diff
@@ -446,7 +457,25 @@ exports[`plugins/MainMenuActions should match snapshot in web view 1`] = `
           ]
         }
         teamId="someteamid"
-      />
+      >
+        <MenuItemToggleModalRedux
```

Comment:
mmm, not sure, let me verify that

Reply:
no unintended change, just removing previously required props used for the previous next steps. It appeared because I added this value that TS was complaining for it not to be present on the test: https://github.com/mattermost/mattermost-webapp/pull/10187/files#diff-1c5f991b9fcdf25fe33e9a38571dcf9a5ebdd1df67f58dedefa1e2afbb264222R34